exporter.py

```diff
@@ -93,12 +93,16 @@ def parse_config(args):
         result = dict(config.items('gtimelog_exporter'))
         mandatory_fields = [
             'jira_url',
-            'tempo_api',
-            'tempo_user',
-            'odoo_url',
-            'odoo_db',
-            'odoo_user'
+            'tempo_url',
+            'jira_account_email',
         ]
+
+        if not (args.no_attendance or result.get('no_attendance')):
+            mandatory_fields.append([
```

Comment:
in my case `mandatory_fields.append` returns an error below when you do `set(mandatory_fields)` with `TypeError: unhashable type: 'list'` . I propose to use `mandatory_fields.extend` instead.

Reply:
in fact the odoo part is not needed anymore... or at least we are not using it anymore since quite some time. Worth to keep it?